vectordb_bench/backend/clients/redis/config.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import SecretStr
+from ..api import DBConfig
+
+class RedisConfig(DBConfig):
+    name: str = "" 
+    password: str = SecretStr
+    host: str = ""
```

Comment:
Please refer to https://github.com/zilliztech/VectorDBBench/blob/675b48f0d2b5561886af17dd897d789bd5e76f5b/vectordb_bench/backend/clients/qdrant_cloud/config.py#L8

SecretStr is a type not a value. Please change host to SecretStr type too. These are both sensitive info.